### ISSUE TITLE
Modal title is missing!

### ISSUE THREAD

**Issue by mosaeedi** [GeneralMagicio/pw-frontend#83]
<img width="616" alt="Screenshot 2023-10-18 at 3 43 52 PM" src="https://github.com/GeneralMagicio/pw-frontend/assets/13708322/62a06719-893a-42b7-b85a-e949f9b20d62">

The **Question** title is missing

**Comment by mmahdigh** [comment]
@MoeNick @Zeptimus

Please check this and #78, as I remember that we decided at the moment to remove the `Question` header for these two instances.

**Comment by MoeNick** [comment]
Yes, I was thinking Mo means the formatting (front and b and size) should be like design, but let's keep Question title removed
